rele/client.py

```diff
@@ -50,7 +50,10 @@ def __init__(self):
     def publish(self, topic, data, blocking=False, **attrs):
         payload = JSONRenderer().render(data)
         logger.info(f'Publishing to {topic}',
-                    extra={'pubsub_publisher_attrs': attrs})
+                    extra={'pubsub_publisher_attrs': attrs,
+                           'metrics': self._build_metrics(topic),
```

Comment:
This indentation should be same style as the way you've done in `_build_metrics`

Reply:
Fix indentation